cognite/client/_api/files.py

```diff
@@ -36,6 +40,7 @@ def __call__(
         data_set_ids: List[int] = None,
         data_set_external_ids: List[str] = None,
         labels: LabelFilter = None,
+        geo_location: GeoLocationFilter = None,
```

Comment:
nit: Mabe adding a new argument at the end of the argument list maybe more suitable for better backwards compatibility? Similarly for next two changes.

Reply:
No particular reason. I think I added `labels` right below where `data_set_ids` were added, not sure why they were added in the middle and not on the bottom. And then now I added `geo_location` right below where I added `labels`.

Shouldn't matter when named arguments are used though?

Reply:
Yup.